src/main/java/uk/nhs/digital/nhsconnect/lab/results/model/edifact/RangeDetail.java

```diff
@@ -0,0 +1,74 @@
+package uk.nhs.digital.nhsconnect.lab.results.model.edifact;
+
+import lombok.AllArgsConstructor;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.EdifactValidationException;
+import uk.nhs.digital.nhsconnect.lab.results.model.edifact.message.Split;
+
+import java.math.BigDecimal;
+
+/**
+ * Examples:
+ *
+ * {@code RND+U+170+1100'}: between 170 and 1100<br/>
+ * {@code RND+U++999'}: less than 999<br/>
+ */
+@EqualsAndHashCode(callSuper = true)
```

Comment:
I believe `callSuper` should be set to `false` as it'll use the superclass's versions of `.equals` and `.hashCode` which will compare the fields of `Segment` in the generated `.equals` and `.hashCode` methods for this class

Reply:
It should be false - my mistake.
Not quite right about the explanation though. With `callSuper = true`, the generated `.equals()` and `.hashCode()` methods use `super.equals()` and `super.hashCode()` _as part of_ the processes; with it `false` it just skips that step.

In our case, since we don't have an explicit `equals` or `hashCode` in `Segment` it uses the default `Object` implementations which rely on instance identity. We set it to `false` because we want to allow different objects [in memory] with identical state to compare as being the same thing.